firefox_ui_tests/functional/locationbar/test_favicon_in_autocomplete.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestFaviconInAutocomplete(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Location bar suggests "History"
+        self.prefs.set_pref('browser.urlbar.default.behavior', 1)
```

Comment:
This pref is not used anymore. You should make use of one of the newer ones similar to https://github.com/mozilla/firefox-ui-tests/blob/master/firefox_ui_tests/functional/locationbar/test_suggest_bookmarks.py.